src/test/java/io/pravega/connectors/flink/utils/SetupUtils.java

```diff
@@ -234,7 +262,7 @@ public void createTestStream(final String streamName, final int numSegments)
         ClientConfig getClientConfig();
     }
 
-    static class InProcPravegaGateway implements PravegaGateway {
+    class InProcPravegaGateway implements PravegaGateway {
```

Comment:
Can we have a separate Gateway for secure cluster `SecureInProcPravegaGateway extends InProcPravegaGateway` and use this for running the secure tests?

Reply:
There seems no need for that as InProcPravegaCluster now requires builder to provide the secure related parameters, user/password, key file, etc. Otherwise, it would throw NPE.

I will take a look at this again to see if it make senses in the context of reverting SetupUtils back to static.